index.html

```diff
@@ -329,6 +329,64 @@ <h3>Create better cloud outcomes with hands-on experience</h3>
             </div>
         </div>
     </section>
+    <!--keep uptodate section-->
+    <section class="keep-up-to-date-section">
+
+        <div class="keep-up-to-date-heading">
+            <h6 class="title-pink">KEEP UP TO DATE</h6>
+            <h3 class="title-sub">Discover more about Pluralsight</h3>
+        </div>
+        <div class="social-items-all">
+            <div class="social-item">
+                <div>
+                    <img class="social-image-size"
+                        src="https://www.pluralsight.com/content/pluralsight/en/jcr:content/main/generic_block_135160/parsys/columns/column-parsys-1/flex_block/parsys/flex_open_block_copy_383891725/parsys/image/image-res.img.5eb03cae-1f8e-4ae5-bcd5-e583792df242.jpg"
+                        alt="">
+                </div>
+                <div class="">
```

Comment:
Do you know what's wrong here?

Reply:
Yes ..deleted the class syntax .but those div are needed right?

Reply:
to make this block level element